operations/network/dns/PowerDNS_Troubleshooting.md

```diff
@@ -0,0 +1,141 @@
+# PowerDNS troubleshooting
+
+## List DNS zone contents
+
+The PowerDNS zone database is populated with data from two sources
+
+* The cray-powerdns-manager service creates the zones and DNS records based on data sourced from the System Layout Service (SLS)
+* The external DNS records are populated by the cray-externaldns-external-dns service using data sourced from Kubernetes annotations and virtual service definitions.
+
+The `cray-powerdns-visualizer` command can be used to view the zone structure that cray-powerdns-manager will create.
```

Comment:
Change to "Use the `cray-powerdns-visualizer` command to view the zone structure that cray-powerdns-manager will create."

Reply:
Fixed